### ISSUE TITLE
This thing crashes whenever I load it. 

### ISSUE THREAD

**Issue by happymimimix** [devellison/pianoshooter#1]

**Comment by happymimimix** [comment]
![image](https://github.com/devellison/pianoshooter/assets/107282563/a4112c0b-3200-4340-9db0-34592c5a21dd)

**Comment by devellison** [comment]
Been a while (!) since I worked on this, but loaded up some old projects with PianoShooter in the latest FL Studio (21.2.2) as well as trying PianoShooter in a blank project with just ZGameEditor and it loaded up fine.  

- What version of FL Studio are you running?  
- If you add ZGameEditorVisualizer to a blank project in FL and drop in PianoShooter, does it work then? 
- Are you using multiple plugins from the ImpostoremTools, or just PianoShooter?
- Which version of PianoShooter is it?  
  - Latest from `main`?  
  - Most recent Release (1.1.1)?
  - 1.0?  
- If you're not sure on version, what's the filesize of PianoShooter.zgeproj?

Basically I'll need to figure out how to recreate this problem, so any additional details would be helpful.  A minimal project it happens in might be extremely helpful.

Thanks!

**Comment by alex-smitg** [comment]
Same. I think it's just not supported in FL Studio 12.

**Comment by devellison** [comment]
Ah! OK, thanks for the clue!
Tested in FL Studio 12, and you are correct.  There's an attribute mismatch that causes a crash there after it fails to load.
Should work in FL Studio 20 and up, tested and verified in 21-25 so far.
I'm swamped right now, but if I get a chance to circle back I'll look at that.
